metis-authentication/metis-authentication-service/src/main/java/eu/europeana/metis/authentication/service/AuthenticationService.java

```diff
@@ -45,6 +48,10 @@
   private static final int CREDENTIAL_FIELDS_NUMBER = 2;
   private static final String ACCESS_TOKEN_CHARACTER_BASKET = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
   private static final int ACCESS_TOKEN_LENGTH = 32;
+  private static final Pattern TOKEN_MATCHING_PATTERN = Pattern
+      .compile("^[" + ACCESS_TOKEN_CHARACTER_BASKET + "]*$");
```

Comment:
Is this not the same as `^[a-zA-Z0-9]*$`?

Reply:
Indeed, I guess I did an automatic conversion from the string comparison.

Reply:
Ah, i guess I didn't see that the BASKET is used outside a regex setting. THen perhaps it is better to leave it as it is?

Reply:
I couldn't find another way to use a regex to generate the tokens, so leaving it as is.